src/modules/units/components/Units.tsx

```diff
@@ -15,10 +15,6 @@ import UnitGroupFormDialog from '@/modules/units/components/UnitGroupFormDialog'
 import UnitManagementTable from '@/modules/units/components/UnitManagementTable';
 import { useGetProjectUnitGroupsQuery } from '@/types/gqlTypes';
 
-// This page has 2 "modes" based on whether the project supports Coordinated Entry referrals.
```

Comment:
This comment was already outdated, should have been removed in https://github.com/greenriver/hmis-frontend/pull/1253